roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -102,20 +106,12 @@ def local_images(self, images, task_vars):
 
     def is_image_local(self, image, task_vars):
         result = self.module_executor("docker_image_facts", {"name": image}, task_vars)
-        if result.get("failed", False):
```

Comment:
The module returns an empty list for the `images` key if it is unable to fetch info about a given image. This evaluates to `False` anyway, so I deemed this branch to be redundant